src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/CipherData.cs

```diff
@@ -0,0 +1,127 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Xml;
+
+namespace System.Security.Cryptography.Xml
+{
+    [System.Security.Permissions.HostProtection(MayLeakOnAbort = true)]
```

Comment:
I can go ahead and do this for HostProtection, SecuritySafeCritical and ComVisible  it could be a community item but I may as well to minimize noise during further reviews. I will keep RegistryPermission however for those that clean up the registry access.